### PR TITLE
Handle ports > 32k in Zipkin JSON

### LINKED ISSUE #467: Zipkin v1 endpoint uses int16 for port, cannot represent ports > 32k
My docker network granted me ports in the range of 56000, which used as ports in endpoint cannot be unmarshalled by the Zipkin Endpoint, because it defines int16 for port field.

### PR DESCRIPTION
**Pull request by yurishkuro**
Resolves #467

### CODE REVIEW

File: cmd/collector/app/zipkin/http_handler_test.go [hunk 1]
```diff
@@ -83,20 +83,23 @@ func TestViaClient(t *testing.T) {
...
+
+func waitForSpans(t *testing.T, handler *mockZipkinHandler, expecting int) {
 	deadline := time.Now().Add(2 * time.Second)
 	for {
 		if time.Now().After(deadline) {
 			t.Error("never received a span")
 			return
 		}
-		if want, have := 1, len(handler.zipkinSpansHandler.(*mockZipkinHandler).getSpans()); want != have {
+		if want, have := expecting, len(handler.getSpans()); want != have {
```
Comment (black-adder) [inline cmd/collector/app/zipkin/http_handler_test.go]: is "want" necessary? just do 

`if have := ....; expected != have {`

File: cmd/collector/app/zipkin/json.go [hunk 2]
```diff
@@ -147,10 +147,15 @@ func endpointToThrift(e endpoint) (*zipkincore.Endpoint, error) {
 	if err != nil {
 		return nil, err
 	}
+	port := e.Port
+	if port >= (1 << 15) {
+		// Zipkin.thrift defines port as i16, so values between (2^16 and 2^16-1) must be encoded as negative
```
Comment (black-adder) [inline cmd/collector/app/zipkin/json.go]: 2^15 and 2^16-1?

Comment (yurishkuro) [reply cmd/collector/app/zipkin/json.go]: what's the question?

Comment (yurishkuro) [reply cmd/collector/app/zipkin/json.go]: oh, yeah

**Pull request merged**